pkg/auth/auth.go

```diff
@@ -170,6 +170,11 @@ func (u *UserAuth) isAllowed(verb string, itemsToCheck []resource) ([]Action, er
 	for _, i := range itemsToCheck {
 		rInfo, err := u.resolve(i.APIVersion, i.Kind)
 		if err != nil {
+			if k8sErrors.IsNotFound(err) {
+				// The resource version/kind is not registered in the k8s API so
+				// we assume it's a CRD that is going to be created with the chart
```

Comment:
There is a case here where a chart tries to install a resource that doesn't exist, I would point out in this comment that it's okay to ignore it here as the actual chart installation will fail in this case